node_modules/col-assets/lib/api.js

```diff
@@ -222,22 +228,118 @@ var createLink = module.exports.createLink = function(ctx, title, url, opts, cal
     return callback({'code': 400, 'msg': validationResult.error.details[0].message});
   }
 
-  // Ensure that all provided categories exist
-  CategoriesAPI.getCategoriesById(ctx, opts.categories, function(err, categories) {
+  // Create the link asset in the DB
+  var asset = {
```

Comment:
Can you put the temporary workaround for link `thumbnail_url`s back in this function instead of the `createAsset` function?